Floyd's_Cycle_Detection_Algorithm/Floyd's_Cycle_Detection.cpp

```diff
@@ -0,0 +1,51 @@
+#include<bits/stdc++.h>
+using namespace std;
+ 
+// Structure of Linked List node
+struct Node 
+{ 
+    int data; 
+    Node* next; 
+};
+Node *head = NULL;
+
+void insert(int value) 
+{ 
+    // Create a new node and adjusting links
+    Node* temp = new Node;
+    temp->data = value; 
+    temp->next = head; 
+    head = temp; 
+} 
+
+ // Function to detect cycle 
+bool detectCycle() 
+{ 
+    Node *slow_ptr = head;
+    Node *fast_ptr = head;   
+    while (slow_ptr!= NULL && fast_ptr!=NULL && fast_ptr->next != NULL) 
```

Comment:
no need for the unnecessary condition (slow_ptr!= NULL). fast_ptr will already take care of it.

Reply:
Removed redundant checks @gupta2140